core/src/services/onedrive/backend.rs

```diff
@@ -0,0 +1,212 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
QQ: Do we have a local command to add this header to new files automatically?

Reply:
`docker run -it --rm -v $(pwd):/github/workspace ghcr.io/korandoru/hawkeye-native format`

We are using this tool to do such work.

Reply:
Got it! Thanks!